invokeai/backend/stable_diffusion/diffusers_pipeline.py

```diff
@@ -439,6 +445,74 @@ def generate_latents_from_embeddings(
         if timesteps.shape[0] == 0:
             return latents, attention_map_saver
 
+        # print("ip_adapter_image: ", type(ip_adapter_image))
+        if ip_adapter_data is not None and len(ip_adapter_data) > 0:
+            ip_adapter_info = ip_adapter_data[0]
+            ip_adapter_image = ip_adapter_info.image
+            # initialize IPAdapter
+            print("   width:", ip_adapter_image.width, " height:", ip_adapter_image.height)
```

Comment:
Many new `print()` statements here

Reply:
Apologies, I left on vacation before I had a chance to clean up all my print diagnostics!